rdfdesc/description.vsp

```diff
@@ -184,7 +184,19 @@ again:
       all_langs := vector (lang_parm, 1.0);
       langs := lang_parm;
     }
-  gr := get_keyword ('g', params, 'unknown');
+
+  -- all possible inputs for IRI to be desribed  
```

Comment:
Typos are one reason why I suggest that PRs only be merged after a reasonable window for review by other eyes...
```suggestion
  -- all possible inputs for IRI to be described  
```

Reply:
moved to separate PR